src/main/webapp/src/assets/styles/styles.scss

```diff
@@ -14,3 +14,6 @@
     margin: 0;
     padding: 0;
 }
+html {
```

Comment:
Please don't just add space globally to the whole html

Reply:
since we do not have footer we have to centerlize a space in the bottom. If we do not centerlize it then we have to add it in each component.

Reply:
Sounds to me like we should add an empty footer component then :)
But adding to globally to the html just screams for issues later on

Reply:
I can do an empty footer container but I am not sure we will need it in the future.

Reply:
I would prefer that. Even if it has no real use, I think it's still better then adding the space to the html since that would make it more difficult to work around if we don't want to have that space somewhere else :)

Reply:
done :+1: